validadteCardNumber_Andrei.js

```diff
@@ -0,0 +1,68 @@
+const cardTypes = {
+  amex: {
+    name: 'AMEX',
+    inicialDigits: [34,37],
+    length: 15
+  },
+  discover: {
+    name: 'Discover',
+    inicialDigits: 6011,
```

Comment:
Por uma questão de consistência, eu manteria `initialDigits` e `length` como sendo sempre de um mesmo tipo de dados (array, neste caso), independentemente da quantidade de itens deste array. Isso remove a necessidade de fazer checks com `typeof` e também de termos problemas de funções indefinidas. `forEach is not a function`, por exemplo.